Silicon/TigerlakePkg/Include/GlobalNvsAreaDef.h

```diff
@@ -0,0 +1,26 @@
+/**@file
+
+  Copyright (c) 2013 - 2017, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+#ifndef __GLOBAL_NVS_AREA_DEF_H__
+#define __GLOBAL_NVS_AREA_DEF_H__
+
+#include <PlatformBase.h>
+#include <PlatformNvsAreaDef.h>
+#include <PchNvsAreaDef.h>
+#include <CpuNvsAreaDef.h>
+#include <SaNvsAreaDef.h>
+
+#pragma pack (push,1)
+typedef struct {
+  PLATFORM_NVS_AREA     PlatformNvs;
+  PCH_NVS_AREA          PchNvs;
+  CPU_NVS_AREA          CpuNvs;
+  // todo: Restricted

```

Comment:
what's this meaning? maybe just remove the comments if it is not used.

Reply:
Noted, I will remove the //todo comment.